packages/core/tests/wallet.test.ts

```diff
@@ -101,4 +102,39 @@ describe('=== wallet', () => {
 
     await expect(aliceAgent.wallet.open(walletConfig)).resolves.toBeUndefined()
   })
+
+  test('when exporting and importing a wallet, content is copied', async () => {
+    await bobAgent.initialize()
+    const bobBasicMessageRepository = bobAgent.injectionContainer.resolve(BasicMessageRepository)
+
+    const basicMessageRecord = new BasicMessageRecord({
+      id: 'some-id',
+      connectionId: 'connId',
+      content: 'hello',
+      role: BasicMessageRole.Receiver,
+      sentTime: 'sentIt',
+    })
+
+    // Save in wallet
+    await bobBasicMessageRepository.save(basicMessageRecord)
+
+    if (!bobAgent.config.walletConfig) {
+      throw new Error('No wallet config on bobAgent')
+    }
+
+    const backupKey = 'someBackupKey'
+    const backupWalletName = `backup-${uuid()}`
+    const backupPath = path.join(tmpdir(), backupWalletName)
+
+    // Create backup and delete wallet
+    await bobAgent.wallet.export({ path: backupPath, key: backupKey })
+    await bobAgent.wallet.delete()
+
```

Comment:
Might be quite redundant but could we add an `expect` to see that the messageRecord does not exist after deletion?